### ISSUE TITLE
Is changing the meaning of negative Order's acceptable?

### ISSUE THREAD

**Issue by ChrisLusena** [nunit/nunit#3892]
There is often a request and question about how to get a test to run last in NUnit, for instance  #3675  or on [stack overflow](https://stackoverflow.com/questions/23688005/how-to-force-a-test-run-last-in-visual-studio-using-nunit)  I think  this can be done if one rewrites the WorkItemOrderComparer to make negative number come last i.e. -1 is the lastest, -2 just before that..  
I don't know how this would break things for people in general though. Are there regression test I need to consider? 

Alternatively one could dive into the engine and allow for an injection of different WorkItemOrderComparer somewhere., though I don't understand that part of NUnit to make a proposal yet. 

Thanks.

**Comment by rprouse** [comment]
I can understand the need, but the workaround of using very large numbers for those tests is trivial, so I'd rather not complicate the code or introduce a possibly breaking change.

**Comment by mikkelbu** [comment]
I think @ChrisLusena wants to run the tests with negative numbers *after* the tests with no `OrderAttribute`. Right now we will run all the ordered tests before the unordered.

**Comment by ChrisLusena** [comment]
@mikkelbu   Yes and  I think other people have this issue.  My current case is that I am testing a data set, where for some cases I cannot build / activating  the data set in the TestFixture constructor nor the set up, so preliminary "tests" are used as part of the set up process.  Then I have a bunch of test who's order does not matter, but finally I have a set of tests that make sure that certain test where run.  Lot of reflection and data inspection to reduce the code size.  Ideally for cleanliness I would like the prelim test to be 1,2,3 and finally meta-tests -2, -1 and all the ones that don't mater un-ordered.  The easy change is breaking.  @rprouse  said.  There are a work arounds, but requires diligence and coordination on the large test fixtures.

**Comment by rprouse** [comment]
I'm still not sure I am convinced that we should add this, but that does make more sense. I am reopening as an idea for discussion.
